src/handlers.rs

```diff
@@ -72,6 +72,12 @@ use {
     bdk_wallet::chain::{BlockId, CanonicalizationParams, CheckPoint},
 };
 
+/// Well-known unspendable key used for Taproot descriptors when only script path is intended.
+/// This is a NUMS (Nothing Up My Sleeve) point that ensures the key path cannot be used.
```

Comment:
since you provided detailed comments in the usage block, this documentation seems redundant.